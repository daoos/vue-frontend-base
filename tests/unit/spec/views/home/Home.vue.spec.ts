import { HomeFactory } from './HomeFactory'

describe('Pagina Home', () => {
  const factory = new HomeFactory()

  beforeEach(async (done) => {
    await factory.montarComponente()
    done()
  })

  it('Dado click no botão dialogo, o dialogo deverá ser exibido', async (done) => {
    await factory.dadoClickNoBotaoDialogo()

    expect(factory.oDialogoEstaAberto()).toBeTruthy()
    done()
  })

  it('Snapshot', async (done) => {
    expect(factory.componente.html())
      .toMatchSnapshot()
    done()
  })
})
