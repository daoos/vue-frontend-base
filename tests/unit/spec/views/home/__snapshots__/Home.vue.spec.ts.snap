// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pagina Home Snapshot 1`] = `
<v-container grid-list-xs="">
  <v-layout justify-center="" mt-4="">
    <v-btn id="btn-dialogo" color="success">Dialogo</v-btn>
  </v-layout>
</v-container>
`;
